RELEASE_NOTES.md

```diff
@@ -1,11 +1,11 @@
 ## Enhancements
-
 - [KOGITO-2562](https://issues.redhat.com/browse/KOGITO-2562) - Knative Eventing resources created for Kogito Applications that uses [Knative Addon](https://github.com/kiegroup/kogito-examples/tree/stable/process-knative-quickstart-quarkus).
-- [KOGITO-3094](Provide data-index-infinispan/mongodb as independent images) - Provide data-index-infinispan/mongodb as independent images
+- [KOGITO-3094](https://issues.redhat.com/browse/KOGITO-3094) - Provide data-index-infinispan/mongodb as independent images
```

Comment:
@ricardozanini has updated it as well on another PR, maybe remove from here?

Reply:
yup, we should merge mine first